src/label.ts

```diff
@@ -283,6 +255,28 @@ export default class GlideCoreLabel extends LitElement {
     );
   }
 
+  #onLabelMouseOver() {
```

Comment:
We haven't see performance issues with Resize Observer, so in a way this is a premature optimization. But it was also excessive for Label to be taking measurements before it needed to.